test/tic_tac_toe_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule TicTacToeTest do
+  use ExUnit.Case
+
+  import ExUnit.CaptureIO
+
+  test "starts new game humanVsHuman where X wins and no replay" do
```

Comment:
I would avoid `humanVsHuman` as almost a technical term here.
How about
`test "a game between two humans where X wins without wanting a replay"`?
The same goes for the other two tests.